src/main/resources/db/changelog/changes/12-insert-categories.yaml

```
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-categories
+      author: solomia
+      changes:
+        - insert:
+            tableName: categories
+            columns:
+              - column:
+                  name: name
+                  value: Category 1
+              - column:
+                  name: description
+                  value: Description 1
+        - insert:
+            tableName: categories
+            columns:
+              - column:
+                  name: name
+                  value: Category 2
+              - column:
+                  name: description
+                  value: Description 2
```

always make sure file ends with one empty line
```suggestion
                  value: Description 2

```